### PR TITLE
update Site schema with container app enviornment

### PR DESCRIPTION
**Pull request by kaibocai**
# Data Plane API - Pull Request
add managedEnvironment to Site schema to support project centauri

## API Info: The Basics
Most of the information about your service should be captured in the issue that serves as your [*engagement record*](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/271/Azure-REST-API-Stewardship?anchor=rest-api-stewardship-process).

* Link to engagement record issue:

Is this review for (select one):

- [X] a private preview
- [ ] a public preview
- [ ] GA release

### Change Scope
<sup>This section will help us focus on the specific parts of your API that are new or have been modified. <br/>Please share a link to the design document for the new APIs, a link to the previous Open API document (swagger) if applicable, and the root paths that have been updated. </sup>
* Design Document: https://microsoftapc-my.sharepoint.com/:w:/g/personal/raorugan_microsoft_com/EQW2cBdgC0tFnbM6x_0XOygBzPr3P--dbbZ6TDdYKMACNg
* Previous Open API Doc:
* Updated paths:

## ❔Got questions? Need additional info?? We are here to help!

<details>
  <summary> Contact us!</summary>

The [Azure API Review Board](https://aka.ms/azapi) is dedicated to helping you create amazing APIs. You can read about our mission and learn more about our process on our [wiki](https://aka.ms/azapi).
* 💬 [Teams Channel](https://teams.microsoft.com/l/channel/19%3a3ebb18fded0e47938f998e196a52952f%40thread.tacv2/General?groupId=1a10b50c-e870-4fe0-8483-bf5542a8d2d8&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47)
* 💌 [email](mailto://azureapirbcore@microsoft.com)

</details>

<details>
  <summary>Click here for links to tools, specs, guidelines & other good stuff</summary>

### Tooling
 * [Open API validation tools](https://aka.ms/swaggertools) were run on this PR. Go here to see [how to fix errors](https://aka.ms/ci-fix)
 * [Spectral Linting](https://aka.ms/style)
 * [Open API Hub](https://aka.ms/openapiportal)

### Guidelines & Specifications
 * [Azure REST API Guidelines](https://aka.ms/guidelines)
 * [OpenAPI Style Guidelines](https://aka.ms/style)
 * [Azure Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy)

### Helpful Links
 * [Azure DevTools Wiki](https://aka.ms/azapi)

</details>

### CODE REVIEW

**Review by kamperiadis — APPROVED** [review]
LGTM! Just one quick question - what is the custom-words.txt file used for?

File: specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/CommonDefinitions.json [hunk 1]
```diff
@@ -3078,6 +3078,10 @@
             "virtualNetworkSubnetId": {
               "description": "Azure Resource Manager ID of the Virtual network and subnet to be joined by Regional VNET Integration.\nThis must be of the form /subscriptions/{subscriptionName}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualNetworks/{vnetName}/subnets/{subnetName}",
               "type": "string"
+            },
+            "managedEnvironment": {
+              "description": "containerapp environment ID. It is used to indentify customer's kube/centauri environment",
```
Comment (jocawtho) [inline specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/CommonDefinitions.json:3083]: Azure Resource Manager ID of the customer's selected Managed Environment on which to host this app. This must be of the form /subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.App/managedEnvironments/{managedEnvironmentName}

**Review by weidongxu-microsoft — APPROVED** [review]

**Comment by kaibocai** [comment]
> LGTM! Just one quick question - what is the custom-words.txt file used for?

I think it's used for one of the pipeline test check, seems it restrict to use words only in this file. 
Anyway I remove my change on this file with the new description.

File: specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/CommonDefinitions.json (hunk 1 above)
Comment (jocawtho) [reply specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/CommonDefinitions.json]: Spoke to Navy. To maintain convention. Lets also call this managedEnvironmentId

**Review by edwin-msft — APPROVED** [review]

**Review by edwin-msft — APPROVED** [review]

**Pull request merged**